spec/tom_queue/tom_queue_integration_spec.rb

```diff
@@ -342,7 +342,7 @@ def start!
       consumers.each do |c|
         total_size += c.work.size
         c.work.each do |work|
-          work.received_at.should < (work.run_at + 1.0)
+          work.received_at.should < (work.run_at + 2.0)
```

Comment:
"Fixes"  the occasional time out error

Reply:
:car::dash: (👋🏻)

I think you can lean on the `be_within` matcher for this. `work.received_at.should be_within(2.0).of(work.run_at)`. (Might need to sprinkle `to_i` if those are time objects, can't remember if that compares nicely.)

Reply:
Oh hey there, @caius! Fancy seeing you here! `be_within` is much nicer 🎉  I'll get that changed up.